anisoap/representations/ellipsoidal_density_projection.py

```diff
@@ -658,3 +665,87 @@ def transform(self, frames, show_progress=False, normalize=True, rust_moments=Tr
             return normalized_features
         else:
             return features
+
+    def power_spectrum(self, ell_frames, AniSOAP_HYPERS=None, sum_over_samples=True):
```

Comment:
```suggestion
    def power_spectrum(self, ell_frames, sum_over_samples=True):
```
We don't need the `AniSOAP_HYPERS` kwarg because `power_spectrum` is part of the `EllipsoidalDensityProjection` class and therefore already has all that information stored.